packages/core/src/integration-helpers.tsx

```diff
@@ -18,6 +18,7 @@ export const createIconComponentFromSimpleIconsSvgPath: (
     xmlns="http://www.w3.org/2000/svg"
     fill={color}
     viewBox="0 0 24 24"
+    height="100%"
```

Comment:
@todo add comment that this is required for FF & Safari to properly render SVGs. Alternative: via CSS